lib/active_force.rb

```diff
@@ -5,11 +5,13 @@
 require 'active_force/query'
 
 module ActiveForce
-
   class << self
     attr_accessor :sfdc_client
+
+    def sf_api_version
```

Comment:
This is needed to construct the subrequest URL for a single composite operation.  It isn't strictly needed for this feature, but will be useful for implementing other Composite API features.